pkg/app/app.go

```diff
@@ -20,6 +20,9 @@ func NewCR(name, appName, appVersion, appCatalog string) *applicationv1alpha1.Ap
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: "giantswarm",
+			Annotations: map[string]string{
+				"chart-operator.giantswarm.io/force-helm-upgrade": "true",
```

Comment:
Our operators are stateless so IMO it always makes sense to use force.

If later we create app CRs for stateful apps we might need to make this configurable.

Reply:
Sure, let's keep it on for now.